document/core/appendix/gen-index-instructions.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env python3
+
+import os
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+INDEX_INSTRUCTIONS_RST = os.path.join(SCRIPT_DIR, 'index-instructions.rst')
+
+HEADER = """\
+.. index:: instruction
+.. _index-instr:
+
+Index of Instructions
+---------------------
+"""
+
+COLUMNS = [
+    'Instruction',
+    'Binary Opcode',
+    'Type',
+    'Validation',
+    'Execution',
+]
+
+
+def MathWrap(s, default=None):
+    if s is None:
+        if default is not None:
+            return default
+        else:
+            return ''
+    else:
+        return ':math:`{}`'.format(s)
```

Comment:
If we are using python3 should we start using f-strings?  `{s}`

Reply:
That makes us require python 3.6, right? I guess that's OK. Do we use them in other Python scripts in WebAssembly org?

Reply:
[Bikeshed requires 3.7](https://github.com/tabatkins/bikeshed/issues/321) so the version shouldn't be a problem.

Reply:
OK, updated to use f-strings.